db/column_family.cc

```diff
@@ -391,6 +391,11 @@ void SuperVersion::Init(MemTable* new_mem, MemTableListVersion* new_imm,
   refs.store(1, std::memory_order_relaxed);
 }
 
+SequenceNumber SuperVersion::GetEarliestMemTableSequenceNumber() const {
+  SequenceNumber smallest_seqno = std::min(mem->GetEarliestSequenceNumber(), imm->GetEarliestSequenceNumber(false));
```

Comment:
For naming consistency, can you name this variable `earliest_seqno`?

Reply:
Ok. Maybe I should `grep` all world like `smallest_seqno`